src/ct/ct_actions_find.cc

```diff
@@ -139,7 +139,7 @@ void CtActions::find_in_multiple_nodes()
         if (not entry_predefined_text.empty()) {
             _s_options.str_find = entry_predefined_text;
         }
-        title = _s_state.replace_active ? _("Replace in Multiple Nodes") : _("Find in Multiple Nodes");
+        title = _s_state.replace_active ? _("Replace in All Nodes...") : _("Search in All Nodes...");
```

Comment:
this change is incorrect, because you can select either all the nodes or only the selected node and subnodes in the dialog